handlers/aws/replay_trigger.py

```diff
@@ -46,28 +48,28 @@ def get_shipper_for_replay_event(
     if event_input is None:
         raise InputConfigException(f"Cannot load input for input id {event_input_id}")
 
-    output: Optional[Output] = event_input.get_output_by_type(output_type)
+    output: Optional[Output] = event_input.get_output_by_destination(output_destination)
     if output is None:
-        raise OutputConfigException(f"Cannot load output of type {output_type}")
+        raise OutputConfigException(f"Cannot load output with destination {output_destination}")
 
     # Let's wrap the specific output shipper in the composite one, since the composite deepcopy the mutating events
     shipper: CompositeShipper = CompositeShipper()
 
-    if output_type == "elasticsearch":
+    if output.type == "elasticsearch":
```

Comment:
I would suggest to either keep output.type or output_destination common in the whole doc

So output.type to become output.destination

Reply:
`output.destination` is the URL - either the cloud url or ES url. `output.type` can only be `elasticsearch` or `logstash`, they are different things @gizas

Reply:
Ok I had the impression that output.destination does not exist. Thanks for clarification
So destination should become output_url in general to denote the actual info of the variabble. But nw not important